### PR TITLE
RUN-4679 not log password

### PR DESCRIPTION
**Pull request by licui3936**
Tests:  
[Windows 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5bb7ce13cb360141a7dfd026)
[Windows 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5bb7cf0ecb360141a7dfd027)

### CODE REVIEW

**Comment by whyn07m3** [comment]
@licui3936 can it have a test? probably manual one in a "collection"?

File: src/browser/api_protocol/transport_strategy/elipc_strategy.ts [hunk 1]
```diff
@@ -191,7 +191,10 @@ export class ElipcStrategy extends ApiTransportBase<MessagePackage> {
             /* tslint:disable: max-line-length */
             //message payload might contain sensitive data, mask it.
             const disableIabSecureLogging = coreState.getAppObjByUuid(opts.uuid)._options.disableIabSecureLogging;
-            const replacer = (!disableIabSecureLogging && (data.action === 'publish-message' || data.action === 'send-message')) ? this.payloadReplacer : null;
+            let replacer = (!disableIabSecureLogging && (data.action === 'publish-message' || data.action === 'send-message')) ? this.payloadReplacer : null;
```
Comment (HarsimranSingh) [inline src/browser/api_protocol/transport_strategy/elipc_strategy.ts]: Do we need to do this here as well https://github.com/HadoukenIO/core/blob/develop/src/browser/api_protocol/transport_strategy/ws_strategy.ts#L88

Comment (licui3936) [reply src/browser/api_protocol/transport_strategy/elipc_strategy.ts]: I don't see any password logged.  Do we have any login for it? Goldman said no.

Comment (HarsimranSingh) [reply src/browser/api_protocol/transport_strategy/elipc_strategy.ts]: Gotcha. Just noticed it had similar code so wanted to check. 👍

**Pull request merged**